subiquity/client/controllers/progress.py

```diff
@@ -77,7 +77,8 @@ async def _wait_status(self, context):
             try:
                 app_status = await self.app.client.meta.status.GET(
                     cur=self.app_state)
-            except aiohttp.ClientError:
+            except Exception:
+                log.exception("getting status failed")
```

Comment:
Oh wait, this line will be filling up the log :)

Reply:
Let me back up a bit - if catching Exception gets someone stuck, I want a usable log of that event.  What do you think?